apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/QueryOptionsTests.cs

```diff
@@ -40,6 +40,7 @@ public async Task PointReadWithConnectionLevelQueryOptions()
             using (var connection = _fixture.GetConnection())
             {
                 connection.QueryOptions = QueryOptions.Empty.WithOptimizerVersion("1");
+                connection.QueryOptions = connection.QueryOptions.WithOptimizerStatisticsPackage("auto_20191128_14_47_22UTC");
```

Comment:
```suggestion
                connection.QueryOptions = QueryOptions.Empty
                    .WithOptimizerVersion("1")
                    .WithOptimizerStatisticsPackage("auto_20191128_14_47_22UTC");
```

(I think we should use this fluent style ourselves, and encourage it in any samples etc.)